test_runner/batch_others/test_restart_compute.py

```diff
@@ -19,7 +19,7 @@ def test_restart_compute(zenith_cli, pageserver, postgres, pg_bin):
             cur.execute("INSERT INTO foo VALUES ('bar')")
 
     # Stop and restart the Postgres instance
-    pg.stop().start()
+    pg.stop_and_destroy().create_start('test_restart_compute')
```

Comment:
Unrelated change? Extract to separate PR?

Reply:
No, it is related change. Right now we can not restart compute node locally (without page server).